CONTRIBUTING.md

```diff
@@ -44,24 +46,68 @@ go test -timeout=30s -p 1 ./...
 
 `run` and `run-race` targets replace `/usr/sbin/pmm-managed` and restart pmm-managed with `supervisorctl`. As a result, it will use normal filesystem locations (`/etc/victoriametrics-promscrape.yml`, `/etc/supervisord.d`, etc.) and `pmm-managed` PostgreSQL database. Other locations (inside `testdata`) and `pmm-managed-dev` database are used for unit tests.
 
+# Advanced Setup
+## Add instances for monitoring
+`make env-up` just starts the PMM server but it doesn't setup anything to be monitored. We can use [pmm-admin](https://github.com/percona/pmm-admin) and [pmm-agent](https://github.com/percona/pmm-agent) to add instances to be monitored to pmm-managed.
 
-## VSCode
-
-VSCode provides first-class support for devcontainers. See:
-
-* https://code.visualstudio.com/docs/remote/remote-overview
-* https://code.visualstudio.com/docs/remote/containers
+* Clone the pmm-admin [repo](https://github.com/percona/pmm-admin/) and install it by running `make install`.
```

Comment:
I wonder if we can provide another Docker Compose with with all databases (like in pmm-agent repo) and percona/pmm-client Docker image instead of asking _pmm-managed_ contributors to compile pmm-admin and pmm-agent there.

Alternatively, we should have "How to contribute to PMM 2" guide somewhere as opposed to "How to contribute to pmm-managed", and just link to that document there.

/cc @sanjabonic @BupycHuk Thoughts?

Reply:
IMHO we need both...since we treat all this things as their own project we need a "how do I contribute to this thing" and an umbrella "how do we contribute to this product"...then each subset can refer you back to the main "These instructions are for contributing to X a subcomponent of PMM...looking to contribute to PMM as a whole?  See here"